karate-core/src/main/java/com/intuit/karate/core/ScenarioContext.java

```diff
@@ -884,6 +884,13 @@ public void embed(byte[] bytes, String contentType) {
         prevEmbeds.add(embed);
     }
 
+    public void embed(Embed embed) {
```

Comment:
change the above method `public void embed(byte[] bytes, String contentType) {` to call this new one

Reply:
changed!